### DISCUSSION TITLE
Welcome to Ventoy Discussions!

### DISCUSSION THREAD

**Discussion by ventoy** [ventoy/Ventoy#711] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jhelmut** [comment]
I use german win10 and like ventoy very much. But booting img or iso created with WINBUILDER unfortunately leads to an error. I only can
show it in german. "Fehler bei der Initialisierung des interaktiven Anmeldevorgangs". Can anybody help?
Helmut

**Reply by chromer030** [reply]
"Fehler bei der Initialisierung des interaktiven Anmeldevorgangs = The interactive login process failed to initialize

**Comment by midnqp** [comment]
Just a quick question: can I skip building some parts like `Ventoy2Disk.exe`?

**Reply by ventoy** [reply]
Ventoy2Disk.exe will not be rebuilt by default.

**Comment by kjkent** [comment]
@ventoy No real content to post, but I just wanted to say thank you for your work developing Ventoy -- I recently discovered it and it's a solution to a problem I didn't even realise I had! Really incredible stuff; thanks.
